src/lib/viewers/archive/constants.js

```diff
@@ -1,7 +1,7 @@
 const TABLE_COLUMNS = {
-    KEY_MODIFIED_AT: 'key_modified_at',
-    KEY_NAME: 'key_name',
-    KEY_SIZE: 'key_size',
+    KEY_MODIFIED_AT: 'modified_at',
+    KEY_NAME: 'name',
+    KEY_SIZE: 'size',
```

Comment:
What's the purpose of renaming here?

Reply:
Turns out I don't have `sortBy` will still sort properly, probably just a microsecond slower so I changed it back.

Reply:
No it won't be slower because they're exactly the same thing. I rewrite the row data in `getRowData`, so no matter what the key is, you're sorting the same value. One more thing, `[KEY_NAME]` is an object, can the lodash sortBy algorithm sort it well? 
```
[KEY_NAME]: {
    fullPath,
    isExternal: false,
    name,
    type,
}
```

Reply:
We talked off-line about this